Exiled.API/Features/Camera.cs

```diff
@@ -263,6 +263,71 @@ public bool IsBeingUsed
+        /// <returns><see langword="true"/> if <see cref="Camera"/> is not <see langword="null"/>, or <see langword="false"/> if <see cref="Camera"/> is <see langword="null"/>.</returns>
+        public static bool TryGet(Scp079Camera camera, out Camera result) => (result = Get(camera)) != null;
+
+        /// <summary>
+        /// Gets a <see cref="Camera"/> given the specified <paramref name="cameraId"/>.
+        /// </summary>
+        /// <param name="cameraId">The id camera to be shearch</param>
+        /// <param name="result">the result of <see cref="Camera"/>, if <paramref name="cameraId"/> is valid.</param>
+        /// <returns><see langword="true"/> if <see cref="Camera"/> is not <see langword="null"/>, or <see langword="false"/> if <see cref="Camera"/> is <see langword="null"/>.</returns>
+        public static bool TryGet(uint cameraId, out Camera result) => (result = Get(cameraId)) != null;
+
+        /// <summary>
+        /// Gets a <see cref="Camera"/> given the specified <paramref name="cameraName"/>.
+        /// </summary>
+        /// <param name="cameraName">The name of the camera.</param>
+        /// <param name="result">The <see cref="Camera"/>, if <paramref name="cameraName"/> is valid.</param>
+        /// <returns><see langword="true"/> if <see cref="Camera"/> is not <see langword="null"/>, or <see langword="false"/> if <see cref="Camera"/> is <see langword="null"/>.</returns>
+        public static bool TryGet(string cameraName, out Camera result) => (result = Get(cameraName)) != null;
+
+        /// <summary>
+        /// Gets a <see cref="Camera"/> given the specified <paramref name="cameraType"/>.
+        /// </summary>
+        /// <param name="cameraType">The <see cref="CameraType"/> to search for.</param>
+        /// <param name="result">The <see cref="Camera"/> with the given <see cref="CameraType"/>.</param>
+        /// <returns><see langword="true"/> if <see cref="Camera"/> is not <see langword="null"/>, or <see langword="false"/> if <see cref="Camera"/> is <see langword="null"/>.</returns>
+        public static bool TryGet(CameraType cameraType, out Camera result) => (result = Get(cameraType)) != null;
+
+        /// <summary>
+        /// Gets a <see cref="IEnumerable{T}"/> of <see cref="Camera"/> filtered based on a predicate.
+        /// </summary>
+        /// <param name="predicate">The condition to satify.</param>
+        /// <param name="result">A <see cref="IEnumerable{T}"/> of <see cref="Camera"/> which contains elements that satify the condition.</param>
+        /// <returns><see langword="true"/> if <see cref="Camera"/> is not <see langword="null"/>, or <see langword="false"/> if <see cref="Camera"/> is <see langword="null"/>.</returns>
+        public static bool TryGet(Func<Camera, bool> predicate, out IEnumerable<Camera> result)
+        {
+            result = null;
+            if (predicate == null)
+                return false;
```

Comment:
i thinks this shouldn't be needed if predicate is null it's a skill issue from the dev and it's will hit an Exception